src/Type/TypeKind.php

```diff
@@ -6,12 +6,12 @@
 
 class TypeKind
 {
-    const SCALAR         = 0;
-    const OBJECT         = 1;
-    const INTERFACE_KIND = 2;
-    const UNION          = 3;
-    const ENUM           = 4;
-    const INPUT_OBJECT   = 5;
-    const LIST_KIND      = 6;
-    const NON_NULL       = 7;
+    const SCALAR       = 'SCALAR';
+    const OBJECT       = 'OBJECT';
+    const INTERFACE    = 'INTERFACE';
+    const UNION        = 'UNION';
+    const ENUM         = 'ENUM';
+    const INPUT_OBJECT = 'INPUT_OBJECT';
+    const LIST         = 'LIST';
+    const NON_NULL     = 'NON_NULL';
```

Comment:
I wonder what is the reason for this change?

Reply:
This allows reusing the constant values, they are needed in a few places when dealing with introspection.

Also, naming a constant `LIST` is possible in PHP 7+ only, i think. Since we dropped compatibility, there is no reason to keep the inconsistent name.

Reply:
Oh, were constant names also reserved? :D mkay. I wondered the same about `INTERFACE` for a while.